eng/common/config/labels.ts

```diff
@@ -168,9 +180,12 @@ export const AreaPaths: Record<keyof typeof AreaLabels, string[]> = {
   "emitter:json-schema": ["packages/json-schema/"],
   "emitter:protobuf": ["packages/protobuf/"],
   "emitter:openapi3": ["packages/openapi3/"],
+  "openapi3:converter": [],
```

Comment:
Any reason not to point to the openapi3 package? Does that cause conflicts with the emitter label?

Reply:
yeah in theory pr should be auto labeled when files in that folder get modified(though the policy service is kinda broken for this)

I updated to be the folder that contain that code.

If the policy service start working correctly maybe we can reorganize the files to be a clearer